web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala

```diff
@@ -129,27 +135,30 @@ import scala.util.{Failure, Success}
         .disabled(loading)
         .variant(muiStrings.contained)
         .color(Color.primary)
-        .onClick(_ => doLogin())
+        .`type`(muiStrings.submit)
+        //.onClick(_ => doLogin())
     }
 
     // TODO: Use a form to get the enter key submitting the form
+    form(onSubmit := (doLogin(_)))(
     mui
```

Comment:
This seems to have wrong alignment, which reminds me to enable scalafmt check in circleci